leapp/logger/__init__.py

```diff
@@ -52,39 +52,66 @@ def _remote_emit(self, log_data):
 
 
 def configure_logger(log_file=None):
-    global _logger
-    if not _logger:
+    """
+    Configure loggers as per the description below.
```

Comment:
Should be on the same line with """ (https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)